R/class.R

```diff
@@ -0,0 +1,43 @@
+inject_super <- function(fun) {
+  e <- new.env()
+  e$super <- function() {
+    bt <- reticulate::import_builtins()
+    self <- as.list(parent.frame())$self
+    bt$super(self$`__class__`, self)
+  }
+  environment(e$super) <- environment(fun)
+  environment(fun) <- e
+  fun
+}
+
+#' Create a python class
+#' 
+#' @param classname The classname
```

Comment:
Are there any restrictions on what class names are valid?

Reply:
It seems that it's controlled by this: https://docs.python.org/3/reference/lexical_analysis.html#identifiers meaning that almost any string is accepted